packages/manager/src/components/CheckoutBar/CheckoutBar.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import { CheckoutBar } from './CheckoutBar';
+
+describe('CheckoutBar', () => {
+  const defaultArgs = {
+    calculatedPrice: 30.0,
+    children: <div>Child items can go here!</div>,
+    heading: 'Checkout',
+    onDeploy: () => alert('Deploy clicked'),
```

Comment:
could we change onDeploy to vi.fn()
```suggestion
    onDeploy: vi.fn(),
```
and then write a test where when we click the 'Submit' button when it's not disabled, this gets called?

Reply:
Added!